receiver/mongodbatlasreceiver/alerts.go

```diff
@@ -182,30 +182,50 @@ func (a *alertsReceiver) startPolling(ctx context.Context, host component.Host)
 }
 
 func (a *alertsReceiver) retrieveAndProcessAlerts(ctx context.Context) error {
-	var alerts []mongodbatlas.Alert
+	numAlertsProcessed := 0
```

Comment:
Why declare this here, just to pass it in? You can declare it where it's used.

Reply:
I guess this will be removed in favor of max_pages. Was confused if we wanted a per poll max alerts or per project. Let's see how the revisit of that concept goes.

Reply:
This is a good point. I think per-project is a reasonable way to apply the limits, as long as it's documented clearly. The user still has effective control over the overall maximum, because they have control over which projects are collected.